Source/swiftlint/Commands/AutoCorrectCommand.swift

```diff
@@ -76,5 +78,8 @@ struct AutoCorrectOptions: OptionsProtocol {
             <*> mode <| Option(key: "use-tabs",
                                defaultValue: false,
                                usage: "should use tabs over spaces when reformatting")
+            <*> mode <| Option(key: "indent-width",
+                               defaultValue: 4,
+                               usage: "the width to indent lines when reformatting")
```

Comment:
I'm open to suggestions about the `usage` wording. I found it hard to describe without ending up with a tautology.